src/main/java/com/swervedrivespecialties/exampleswerve/util/util.java

```diff
@@ -0,0 +1,30 @@
+package com.swervedrivespecialties.exampleswerve.util;
+
+public class util {
+    /**
+     * @param minSS The minimum Speed Scale, toggled with ToggleSS
+     * @param additionalSS Additional Speed Scale, added on to minSS
+     * @return Double to use as a scalar for drive velocity
+     */
+    public static double getSpeedScale(double minSS, double additionalSS){
+        return minSS + (1 - minSS) * Math.pow(additionalSS, 2);
```

Comment:
Not on a card right now, but, if it interests you, flipping the sign of the change here is something JZ was interested in doing, so that hitting the trigger acts as a break. You'd probably want to do a sqrt instead of a square, then, but if you want to include that as a mode for the chassis, know that's something to possibly look into.